scripts/install_odf.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+ibmcloud config --check-version=false
+ibmcloud login -apikey ${IC_API_KEY}
+ibmcloud ks cluster config -c ${CLUSTER} --admin
+
+# Retrieve the openshift cluster version
+ROKS_VERSION=`oc get clusterversion -o jsonpath='{.items[].status.history[].version}{"\n"}'`
+# Round the number down to 0 (e.g. 4.7.40 to 4.7.0)
+ROKS_VERSION="${ROKS_VERSION%.*}.0"
+
+ibmcloud oc cluster addon enable openshift-data-foundation -c ${CLUSTER} --version ${ROKS_VERSION} --param "odfDeploy=false"
```

Comment:
If trying to deploy ODF then need to set `odfDeploy=true`
Also there are other parameters of ODF which needs to be taken as input from user and provided here in this command

Reply:
before it was false because odfDeploy was the only parameter being passed. I've added the other parameters so now odfDeploy is set to true